tests/Models/User.php

```diff
@@ -8,6 +8,9 @@
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Tests\Database\Factories\UserFactory;
 
+/**
+ * @phpstan-property bool $active
```

Comment:
@szepeviktor Would there a way for Larastan/PHPStan to automatically detect the `active` model attribute casting into a boolean?

Reply:
Yes!
If there would be 1 migration for the `users` table.
Now there are 2: one in this project, one in Testbench

Reply:
Brute force "solution": remove Testbench's migration before static analysis 💣

Reply:
I leave it up to you after this PR.

Reply:
In fact, there's no need to add `- vendor/orchestra/testbench-core/laravel/migrations/` in `databaseMigrationsPath`, they are not used into the package test suite.

Reply:
It means that we can remove the `@phpstan-property bool $active` dockblock too.

Reply:
Okay-okay.
I'm working overtime :)